src/pages/MyNiwa/index.tsx

```diff
@@ -69,88 +67,79 @@ export const MyNiwa = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [userData]);
 
-  useEffect(() => {
-    getGardenCategories();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
-  const getImage = (categoryId: string) => {
-    const result = categories?.filter(
-      (category) => category._id === categoryId
-    );
-    if (result?.[0]?.imageURL) {
-      return result[0]?.imageURL;
-    }
-    return gardenImage;
-  };
-
-  const history = useHistory();
   const goToCreateGarden = () => {
     history.push("/user/createGarden");
   };
   return (
-    <motion.div
-      initial={{ opacity: 0 }}
-      animate={{ opacity: 1 }}
-      transition={{ duration: 0.3 }}
-      exit={{ opacity: 0 }}
-    >
-      <div className="my-gardens-container">
-        <Grid
-          container
-          className={classes.myNiwaHeader}
-          direction="row"
-          justifyContent="space-between"
-        >
-          <h1>My Niwa</h1>
-          <Tooltip arrow classes={tooltipStyles} title="Add Flower Bed">
-            <IconButton
-              className={classes.createGarden}
-              onClick={goToCreateGarden}
-            >
-              <AddIcon />
-            </IconButton>
-          </Tooltip>
-        </Grid>
-        <LoadingWrapper
-          isLoading={!gardensApi.isLoaded || !categoriesApi.isLoaded}
-        >
-          <div className="gardens-view">
-            {!gardens.length ? (
-              <Typography variant="h4">
-                Use the + button to make a new flower bed! ⤴
-              </Typography>
-            ) : (
-              gardens.map((garden, index) => {
-                return (
-                  <Link to={`/user/dailyGardening/${garden._id}`} key={index}>
-                    <Card className={`garden-card ${classes.root}`}>
-                      <CardActionArea>
-                        <CardMedia
-                          className={classes.media}
-                          image={getImage(garden.gardenCategoryId)}
-                          title="Contemplative Reptile"
-                        />
-                        <CardContent>
-                          <Typography gutterBottom variant="h5" component="h2">
-                            {garden.name}
-                          </Typography>
-                          <Typography
-                            variant="body2"
-                            color="textSecondary"
-                            component="p"
-                          >
-                            {garden.description}
-                          </Typography>
-                        </CardContent>
-                      </CardActionArea>
-                    </Card>
-                  </Link>
-                );
-              })
-            )}
-          </div>
-        </LoadingWrapper>
-      </div>
-    </motion.div>
+    <>
+      <Head />
+      <motion.div
```

Comment:
wrap with fragment and add Head component
and added common component for layout consistency